.github/workflows/c-cpp.yml

```diff
@@ -12,15 +12,15 @@ jobs:
     strategy:
       matrix:
         os:
```

Comment:
Since this commit fixes macos build, maybe it's worth to add mac os as a build target as well?

Reply:
Turns out it wasn't as painful as expected, adding it...